src/odemis/driver/test/actuator_test.py

```diff
@@ -507,6 +523,10 @@ def test_move_abs(self):
         f.result()
         test.assert_pos_almost_equal(stage.position.value, {"x": 1e-06, "y": 2e-06})
         test.assert_pos_almost_equal(child.position.value, {"x":-2e-06, "y": 1e-06})
```

Comment:
question: why are positions for child swapped compared to stage?

Reply:
Because we are testing with rotation = 90°